std/process.d

```diff
@@ -3116,11 +3121,15 @@ unittest
     assertThrown(environment["std_process"]);
 
     // get() without default value
-    assert (environment.get("std_process") == null);
+    assert (environment.get("std_process") is null);
```

Comment:
common usage is to not have the space between `assert` and the `(`; might as well fix it while this line is being changed